src/lib/error-handler/index.ts

```diff
@@ -68,12 +68,12 @@ export const errorHandler = function errorHandler(
     })
   }
   if (err instanceof HttpResponse) {
-    logger.warn('Error thrown in handler: %s', err.message)
+    logger.warn('Error thrown in handler: %s', JSON.stringify(err.message))
```

Comment:
why do we need to stringify here? Could you please give some examples. The message of error instance is a string.
![Screenshot 2023-03-29 at 08 06 24](https://user-images.githubusercontent.com/11794402/228453369-ac6e23f7-10fe-4d07-b391-8728fe250e4c.png)

Reply:
I had some error that was logging as `[Object object]`, I can't remember, it might have been the `dscpApi` error:
```
  const result = await res.json()

  if (res.ok) {
    return result
  }

  throw new HttpResponse({ code: 500, message: result }) // pass through dscpApi error
 ```

Reply:
I'll change `dscp-api` and revert the error handler changes